dev-tools/packer/platforms/centos/beatname.sh.j2

```diff
@@ -1,9 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env
```

Comment:
I think bash is missing here

Reply:
Probably should have copied the file instead of typing it twice.

Reply:
Fixed.